internals/secrethub/service_gcp_init.go

```diff
@@ -48,6 +48,11 @@ func (cmd *ServiceGCPInitCommand) Run() error {
 		return err
 	}
 
+	_, err = client.Repos().Get(cmd.repo.String())
```

Comment:
I think this would benefit from a comment.
```suggestion
	// Fail fast if the repo does not exist.
	_, err = client.Repos().Get(cmd.repo.String())
```